### ISSUE TITLE
add regression tests for floating code

### ISSUE THREAD

**Issue by bayc** [NREL/floris#526]

**Comment by rafmudaf** [comment]
@bayc is this resolved by #518?

**Comment by bayc** [comment]
> @bayc is this resolved by #518?

No, the existing regression tests were updated for the new code format, but regression tests for floating cases were not added and still need to be.
